node/src/components/api_server/rpcs/chain.rs

```diff
@@ -122,23 +134,23 @@ impl RpcWithOptionalParamsExt for GetStateRootHash {
 }
 
 async fn get_block<REv: ReactorEventT>(
-    maybe_hash: Option<BlockHash>,
+    maybe_parameter: Option<BlockParameters>,
     effect_builder: EffectBuilder<REv>,
 ) -> Result<Option<Block>, warp_json_rpc::Error> {
     // Get the block from storage or the latest from the linear chain.
-    let getting_from_storage = maybe_hash.is_some();
+    let getting_from_storage = maybe_parameter.is_some();
```

Comment:
I know this was named `getting_from_storage` before, but it'd be more properly named something like `getting_specific_block` now since all get-block requests go to storage.